src/overlays/actors/ovl_En_Ma3/z_en_ma3.h

```diff
@@ -8,6 +8,29 @@ struct EnMa3;
 
 typedef void (*EnMa3ActionFunc)(struct EnMa3*, GlobalContext*);
 
+typedef enum {
+    /* 0x00 */ MALON_ADULT_LIMB_NONE,
+    /* 0x01 */ MALON_ADULT_ROOT_LIMB,
+    /* 0x02 */ MALON_ADULT_LOWER_CONTROL_LIMB,
+    /* 0x03 */ MALON_ADULT_LEFT_THIGH_LIMB,
```

Comment:
```suggestion
    /* 0x01 */ MALON_ADULT_LIMB_ROOT,
    /* 0x02 */ MALON_ADULT_LIMB_LOWER_CONTROL,
    /* 0x03 */ MALON_ADULT_LIMB_LEFT_THIGH,
```
etc
also I think having the values in decimal rather than hex is more useful

Reply:
Everywhere else its in hex so I'll leave it here.

Reply:
hex: PlayerLimb, EnFuLimb, EnGeldBLimb, EnHataLimb, EnNbLimb, StalfosLimb, EnValiLimb

not hex: BossGomaLimb, EnBiliLimb, EnGe1Limb, EnGomaLimb, EnMbLimb

I guess we can clean this mess up later (for the record I would prefer not hex)